main/src/library/Concurrent/Condition.flix

```diff
@@ -33,7 +33,7 @@ namespace Concurrent/Condition {
     /// Causes the current thread to wait until it is signalled or interrupted.
     ///
     pub def await(condition: Concurrent/Condition.Condition): Result[Unit, ConditionError] & Impure =
-        import java.util.concurrent.locks.Condition.await();
+        import java.util.concurrent.locks.Condition.await(): Unit & Impure;
```

Comment:
Note that the errors shown in the ticket is only the first error per file.

For example, in this file, on line 51 changes must also be made, so you have to go through each import statement manually :)

Reply:
Yes, I'll make sure to get that done. That's why I wanted to know, if the goal was to have type annotation for every import statement :)